### PR TITLE
Feature 996 link migration steps op driver

### PR DESCRIPTION
**Pull request by rajendraindukuri**
# Description
- Added references to the common migration steps page created  wherever there is a deprecation notice
- Updated v1 folder with the same changes as it contains the deprecation notice as well

Most of the original changes done as part of this PR are covered as part of PR https://github.com/dell/csm-docs/pull/887/files
Just keeping the diff after resolving conflicts. We can avoid merging this PR if the diff content need not go into the base branch

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/996|

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by mareksuski-dell — APPROVED** [review]
LGTM

**Review by HarishH-DELL — APPROVED** [review]
LGTM

**Review by chimanjain — APPROVED** [review]

**Review by shefali-malhotra — APPROVED** [review]
LGTM

**Comment by rajendraindukuri** [comment]
Most of the original changes done as part of this PR are covered as part of PR https://github.com/dell/csm-docs/pull/887/files
Just keeping the diff after resolving conflicts. We can avoid merging this PR if the diff content need not go into the base branch.Thanks.

**Review by mareksuski-dell — APPROVED** [review]
LGTM

**Review by shefali-malhotra — APPROVED** [review]
LGTM

**Review by gallacher — CHANGES_REQUESTED** [review]
Fix failing checks

**Comment by shanmydell** [comment]
@gallacher : Failing checks are due to another PR, not related to this PR

**Review by gallacher — APPROVED** [review]

File: content/docs/csidriver/installation/operator/operator_migration.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ description: >
       kubectl -n openshift-operators get CSIUnity/test-unity -o yaml
   ```
 2. Map and update the settings from the CR in step 1 to the relevant CSM Operator CR
-    - As the yaml content may differ, ensure the values held in the step 1 CR backup are present in the new CR before installing the new driver
+    - As the yaml content may differ, ensure the values held in the step 1 CR backup are present in the new CR before installing the new driver. CR Samples table provided above can be used to compare and map the differences in attributes between Dell CSI Operator and CSM Operator CR s
```
Comment (gallacher) [inline content/docs/csidriver/installation/operator/operator_migration.md:33]: At the end of the line "... and CSM Operator CRs"

**Review by shanmydell — DISMISSED** [review]

File: content/docs/csidriver/installation/operator/operator_migration.md (hunk 1 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/operator/operator_migration.md:33]: @rajendraindukuri : Please address above comment

**Review by shanmydell — APPROVED** [review]

**Pull request merged**